tools/automator/automator.sh

```diff
@@ -253,7 +257,7 @@ commit() {
 
   local src_branch="${AUTOMATOR_SRC_BRANCH:-none}"
   fork_name="$src_branch-$branch-$modifier-$(hash "$title")"
-  git -c "user.name=$user" -c "user.email=$email" commit --message "$title" --author="$user <$email>"
+  git -c "user.name=$user" -c "user.email=$email" commit --message "$title" --author="$user <$email>" "${commit_args[@]}"
```

Comment:
Wouldn't this fail (`set -u`) if this variable is not defined?

Reply:
nope, not a problem because it's accessed using `[@]`

Reply:
compare `script_args` and `shell_args` in this same file. they work the same way

Reply:
cool :)